README.md

```diff
@@ -15,9 +15,41 @@ $ make
 $ ./nothing
 ```
 
+## Build on Windows
+
+You need to install [conan][] and [Visual Studio 2017][visual-studio].
+
+### Dependencies
+
+Current version of [SDL2/2.0.5@lasote/stable][conan-sdl2] in the conan-transit
+repository is not compatible with the latest conan, so you'll need to install
+it locally from the git repository:
```

Comment:
How am I supposed to track this technical debt? Do they have an issue for that? I don't see anything in their issue tracker: https://github.com/lasote/conan-sdl2/issues

I see that they fixed that in the source code: https://github.com/lasote/conan-sdl2/pull/13 . Do they have any plans on publishing the latest changes?

Reply:
Actually, doesn't matter. It's just a note in README. Thank you for spending you personal time on this!